aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -448,29 +674,83 @@ def export_tree(
         'input': node_pk_2_uuid_mapping[link.source_id],
         'output': node_pk_2_uuid_mapping[link.target_id],
         'label': link.link_label,
-        'type': link.link_type
+        'type': link.link_type,
     } for link in traverse_output['links']]
 
     progress_bar.update()
 
+    return (
+        node_ids_to_be_exported,
+        node_pk_2_uuid_mapping,
+        links_uuid,
+        graph_traversal_rules,
+    )
+
+
+def _check_node_licenses(
+    node_ids_to_be_exported: Set[int],
+    allowed_licenses: Optional[Union[list, Callable]],
+    forbidden_licenses: Optional[Union[list, Callable]],
+) -> None:
```

Comment:
let's put a docstring

Reply:
fixed in b689f5e6986c15ab18ad225e5659f1ef3bf8f9c0